cmd/buildah/mkcw.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/containers/buildah"
+	"github.com/containers/buildah/define"
+	"github.com/containers/buildah/pkg/parse"
+	"github.com/spf13/cobra"
+)
+
+func mkcwCmd(c *cobra.Command, args []string, options buildah.CWConvertImageOptions) error {
+	ctx := getContext()
+
+	systemContext, err := parse.SystemContextFromOptions(c)
+	if err != nil {
+		return err
+	}
+
+	if options.AttestationURL == "" && options.DiskEncryptionPassphrase == "" {
+		return fmt.Errorf("neither --attestation-url nor --encryption-passphrase flags provided, disk would not be decryptable")
+	}
+
+	store, err := getStore(c)
+	if err != nil {
+		return err
+	}
+
+	options.InputImage = args[0]
+	options.Tag = args[1]
+	options.ReportWriter = os.Stderr
+	imageID, _, _, err := buildah.CWConvertImage(ctx, systemContext, store, options)
+	if err == nil {
+		fmt.Printf("%s\n", imageID)
+	}
+	return err
+}
+
+func init() {
+	var teeType string
+	var options buildah.CWConvertImageOptions
+	mkcwDescription := `Convert a conventional image to a confidential workload image.`
+	mkcwCommand := &cobra.Command{
+		Use:   "mkcw",
+		Short: "Convert a conventional image to a confidential workload image",
+		Long:  mkcwDescription,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			options.TeeType = define.TeeType(teeType)
+			return mkcwCmd(cmd, args, options)
+		},
+		Example: `buildah mkcw localhost/repository:typical localhost/repository:cw`,
+		Args:    cobra.ExactArgs(2),
+	}
+	mkcwCommand.SetUsageTemplate(UsageTemplate())
+	rootCmd.AddCommand(mkcwCommand)
+	flags := mkcwCommand.Flags()
+	flags.SetInterspersed(false)
+
+	flags.StringVarP(&teeType, "type", "t", "", "TEE type")
+	flags.StringVarP(&options.AttestationURL, "attestation-url", "u", "", "attestation server URL")
+	flags.StringVarP(&options.AttestationURL, "attestation_url", "", "", "attestation server URL (alternate flag spelling)")
+	if err := flags.MarkHidden("attestation_url"); err != nil {
+		panic("error marking attestation_url as hidden")
+	}
+	flags.StringVarP(&options.BaseImage, "base-image", "b", "", "alternate base image (default: scratch)")
+	flags.StringVarP(&options.BaseImage, "base_image", "", "", "alternate base image (default: scratch) (alternate flag spelling)")
+	if err := flags.MarkHidden("base_image"); err != nil {
+		panic("error marking base_image as hidden")
+	}
+	flags.StringVarP(&options.DiskEncryptionPassphrase, "encryption-passphrase", "p", "", "disk encryption passphrase")
+	flags.StringVarP(&options.DiskEncryptionPassphrase, "encryption_passphrase", "", "", "disk encryption passphrase (alternate flag spelling)")
```

Comment:
Why not just "passphrase"?

Reply:
Changing it, though it's still not an option I want to encourage people to use.